atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Cell.java

```diff
@@ -116,6 +116,10 @@ public byte[] getColumnName() {
         return columnName;
     }
 
+    public long sizeInBytes() {
+        return Long.sum(rowName.length, columnName.length);
```

Comment:
just curious, is the `Long.sum` method call to implicitly widen and avoid casting to long for addition (e.g. `return ((long) rowName.length) + columnName.length;`)?

Reply:
yes - it also looked more succinct to my eyes